### ISSUE TITLE
How can I change learning rate while finetuning?

### ISSUE THREAD

**Issue by debaser990** [minimaxir/gpt-2-simple#138]
I'm trying to finetune 355M model on my dataset.

**Comment by debaser990** [comment]
Just realized I could pass it as a parameter.
